.nightfalldlp/config.json

```diff
@@ -1,9 +1,41 @@
 {
-    "detectors": [
-        "CREDIT_CARD_NUMBER",
-        "PHONE_NUMBER",
-        "API_KEY",
-        "CRYPTOGRAPHIC_KEY"
+    "conditions": [
+        {
+            "detector": {
+                "detectorType": "NIGHTFALL_DETECTOR",
+                "nightfallDetector": "CREDIT_CARD_NUMBER",
+                "displayName": "cc num"
+            },
+            "minNumFindings": 1,
+            "minConfidence": "VERY_UNLIKELY"
```

Comment:
just curious but is there a reason why we want the confidence to be `VERY_UNLIKELY` for each detector?

Reply:
🤷‍♂️ this file is really just for doing local testing when we push up diffs, so I wanted to see findings regardless of how low their likelihood was. We can bump if needed but I can't imagine us having any issues w/ getting too many findings or something.